### ISSUE TITLE
sudden jump in loss using Tridentnet for Transfer Learning

### ISSUE THREAD

**Issue by mistaleee** [TuSimple/simpledet#196]
Hi all,

like in #70 I get this sudden jump in L1 and L2 loss. Unfortunately, I didn't find a solution in the referenced issue. 

![Bildschirmfoto vom 2019-08-12 15-26-42](https://user-images.githubusercontent.com/24625384/62868634-5f75da00-bd16-11e9-9b25-11ec0f6cbd68.png)

Config changes of tridentnet_r101v2c4_c5_2x (downloaded astridentnet_r101v2c4_c5_multiscale_addminival_3x_fp16.zip):
num_classes = 4
begin_epoch = 18
end_epoch = 20

Dataset: Custom Dataset consisting of 2530 test images and 3531 Train Images, 1843 Val Images (178 minival + 1665 valminusminival)

What causes this jump? 

Thanks for every hint.

**Comment by RogerChern** [comment]
Hi @mistaleee ,
You may need to set a smaller learning rate.

**Comment by mistaleee** [comment]
is the learning rate called lr_factor in detection_train.py?

**Comment by mistaleee** [comment]
ah, got it. The learning rate is calculated by the factor and several other parameters. I just commented out the calculation and set the variable `current_lr` to 0.0001. Training is going on and the losses look good. @RogerChern , thanks again :)

**Comment by mistaleee** [comment]
Unfortunately, there is now another problem regarding the RcnnL1. While the RpnL1 is slowly going down, the RcnnL1 is slowly going up. Now, the RcnnL1 already doubled. What is wrong with my settings?

**Comment by RogerChern** [comment]
Still, this is related to the learning rate. You may need to do a small grid search to find a proper learning rate for your own dataset. The learning rate is quite sensitive to different datasets.

**Comment by mistaleee** [comment]
Do you change the learning rate during training when you train on the coco dataset?

**Comment by RogerChern** [comment]
@mistaleee, yes, the learning rate is divide by 10 twice during the training.

**Comment by RogerChern** [comment]
Close due to inactivity.
